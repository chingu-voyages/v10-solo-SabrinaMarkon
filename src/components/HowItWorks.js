import React, { Component } from 'react';

export default class HowItWorks extends Component {


  render() {
    return (
      <div>
        <h1>HowItWorks</h1>
        <div>
          Lorem ipsum
        </div>
      </div>
    )
  }
}