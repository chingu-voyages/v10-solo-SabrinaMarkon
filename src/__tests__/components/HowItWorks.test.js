import React from 'react';
import { shallow } from 'enzyme';

// components
import HowItWorks from '../../components/HowItWorks';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<HowItWorks />);
});

describe('Testing HowItWorks component', () => {
  it('should render an h1 heading with the correct text', () => {
    expect(wrapper.find('h1').text().toEqual('HowItWorks'));
  });
  it('it should match the HowItWorks snapshot', () => {
    // If snapshot exists, compare. If not, create the snapshot.
    expect(wrapper).toMatchSnapshot();
  });
});