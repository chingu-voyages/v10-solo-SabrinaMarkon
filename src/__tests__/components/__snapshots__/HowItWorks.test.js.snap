// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HowItWorks component it should match the HowItWorks snapshot 1`] = `undefined`;
